httpserver-rs/src/lib.rs

```diff
@@ -149,7 +149,12 @@ impl HttpServerCore {
                       query: String| {
                     let inner = Arc::clone(&inner);
                     let linkdefs = linkdefs.clone();
-                    tracing::span::Span::current().record("query", &tracing::field::display(&query));
+                    //let tracer = opentelemetry::global::tracer("HttpServer");
+                    let span = tracing::debug_span!("http request");
+                    span.record("method", &tracing::field::display(method.to_string().as_str()));
+                    span.record("path", &tracing::field::display(path.as_str()));
+                    span.record("query", &tracing::field::display(query.as_str()));
```

Comment:
Hmm, should this handler anonymous function get a name that we can then `instrument`?

Reply:
maybe, but I currently like the level of verbosity on httpserver traces. I think there may be a way to add those to the debug_span! macro to make it simpler. I'll leave this comment unresolved and come back to it later.

Reply:
(fmt subscribers fixed)

Reply:
Not sure I follow. Do you mean you like seeing these inline over the `instrument` attribute syntax?